packages/upswyng-server/src/components/Autocomplete.svelte

```diff
@@ -0,0 +1,72 @@
+<script context="module">
+  export const key = {};
```

Comment:
I might be missing something -- is the key just an empty obj?

Reply:
oh i saw the docs.. interesting

Reply:
Ya I don't quite understand it myself but it appears to handle managing that itself.